src/classes/UTIL_ACCT_Naming.cls

```diff
@@ -0,0 +1,285 @@
+/*
```

Comment:
@lifewithryan @xinnanlisf When I initially pushed the latest from master into my new scratch org, VS Code (or CCI? not sure which) automatically added this class to the package.xml file. 

Is there a reason it wasn't added to the package.xml, or was it just missed?

I can add it in my branch that I'm working in now, but I wasn't sure how we go about these minor changes here at SFDO.